### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by ClaraFelix**

### CODE REVIEW

File: apps/frontend/src/app/pages/jobs/jobs.component.html
Comment (dev-luizf) [inline apps/frontend/src/app/pages/jobs/jobs.component.html:1]: This loading component isn't being displayed. Also, you don't need all of these conditions, you just need to check @if(isLoading) and@if(!jobs.length) and remove the other ones

**Review by dev-luizf — CHANGES_REQUESTED** [review]
Currently, the loading is infinite on the person pages, and additionally, on the job page, it is appearing along with the message 'there is no content'.

Captura de tela de 2024-05-02 07-37-03

Another detail is that the background of the component is slightly darker than the rest of the page. I believe it should be white.

**Comment by ClaraFelix** [comment]
It's infinite on pages that don't have integration yet, that's why I just called loading

**Comment by ClaraFelix** [comment]
Done :)

**Pull request merged**